docker-for-windows/release-notes.md

```diff
@@ -69,7 +93,7 @@ This page contains information about the new features, improvements, known issue
 ## Docker Desktop 3.3.2
 2021-05-03
 
-> [Download](https://desktop.docker.com/win/stable/amd64/63878/Docker%20Desktop%20Installer.exe)
+> [Download](https://desktop.docker.com/win/stable/amd64/63878/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64)
```

Comment:
The UTM is not needed here.

Reply:
arg, missed this change. I'll update it in a bit